config/crds/pipeline_v1beta1_pipelineparams.yaml

```diff
@@ -22,6 +22,21 @@ spec:
           type: object
         spec:
           properties:
+            clusters:
+              items:
+                properties:
+                  endpoint:
+                    type: string
+                  name:
+                    type: string
+                  type:
+                    type: string
+                required:
```

Comment:
Could `required` be a field on the property?

```
properties:
  endpoint:
    type: string
    required: true
  name:
    type: string
    required: true
  type:
    type: string
    required: true
```

Reply:
This was already there from before this PR, I believe it is added by the `kubebuilder` generation of these crd yaml files

Reply:
Oh oh I see, this is just generated from the types. Sorry, ignore me.